<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drvmng</class>
 <widget class="QMainWindow" name="drvmng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通讯设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>通讯接口类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>CAN</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>LAN</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>设备索引号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CB_Baud">
        <item>
         <property name="text">
          <string>1000K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400K</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>第几路CAN：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_4">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxCount">
         <number>10</number>
        </property>
        <item>
         <property name="text">
          <string>第一路</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>第二路</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>帧类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CB_EXT">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxCount">
         <number>10</number>
        </property>
        <item>
         <property name="text">
          <string>标准帧</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扩展帧</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>IP Address：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IPlineEdit">
        <property name="text">
         <string>192.168.2.208</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Server Port：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PortlineEdit">
        <property name="text">
         <string>9030</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="bt_Connected">
      <property name="text">
       <string>打开设备</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
